projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -362,6 +366,14 @@ function ( $payment_gateway ) {
 	public function order_process( $order_id ) {
 		$order = wc_get_order( $order_id );
 
+		if (
+			! $order
+			|| ! $order instanceof WC_Order
+			|| ! $order instanceof WC_Order_Refund
```

Comment:
How did you figure out about checking for `instanceof WC_Order_Refund` here? Below I see `$order->get_payment_method();`, but I don't see a `get_payment_method()` method for the `WC_Order_Refund` class unless I'm missing something 🤔 

Otherwise, I wasn't able to reproduce the error even trying a dev build of woocommerce-blocks unfortunately, but the PR changes make sense and are consistent with existing checking in the file.